Il2ServerAppLateNight/Il2ServerAppLateNight/OilTemp.cpp

```diff
@@ -13,6 +13,70 @@ bool Intake(std::string planeName)
 	return false;
 }
 
+bool isBF109K4(std::string planeName)
+{
+	std::string v = "Bf 109 K-4";
+	if (planeName.compare(v) == 0)
+		return true;
+
+	return false;
+}
+
+bool isBF110E2(std::string planeName)
+{
+	std::string v = "Bf-110 E2";
+	if (planeName.compare(v) == 0)
+		return true;
+
+	return false;
+}
+
+bool isBF110G2(std::string planeName)
+{
+	std::string v = "Bf-110 G2";
+	if (planeName.compare(v) == 0)
+		return true;
+
+	return false;
+}
+
+bool isHS129B2(std::string planeName)
+{
+	std::string v = "Hs 129 B-2";
+	if (planeName.compare(v) == 0)
+		return true;
+
+	return false;
+}
+
+bool isJU88A4(std::string planeName)
+{
+	std::string v = "Ju-88 A4";
+	if (planeName.compare(v) == 0)
+		return true;
+
+	return false;
+}
+
+bool isJU88C6(std::string planeName)
+{
+	std::string v = "Ju-88 C6";
+	if (planeName.compare(v) == 0)
+		return true;
+
+	return false;
+}
+
+bool isJU52(std::string planeName)
+{
+	std::string v = "Ju-52/3m g4e";
+	if (planeName.compare(v) == 0)
+		return true;
+
+	return false;
+}
+
```

Comment:
These could all be moved to a helper function

Reply:
That would definitely make things tidier